modin/core/storage_formats/base/doc_utils.py

```diff
@@ -567,10 +567,11 @@ def doc_window_method(
     callable
     """
     template = """
-        Create {win_type} and {action} for each window.
+        Create {win_type} and {action} for each window over the given axis.
 
         Parameters
         ----------
+        fold_axis : {{0, 1}}
```

Comment:
Perhaps I'm missing something, but don't see `fold_axis` as a parameter for `doc_window_method`.

Reply:
It's not a parameter to `doc_window_method`, but it's a parameter to each of the window methods in the query compiler :) `doc_window_method` generates the docstrings for those methods. The `fold_axis` line here is in a docstring template and will go into the docstring for each method.